app/lib/constants.js

```diff
@@ -5,4 +5,6 @@ module.exports = {
   numberOfNearbyResultsToDisplay: 3,
   promHistogramBuckets: [0.01, 0.05, 0.1, 0.2, 0.3, 0.5, 1, 1.5, 5, 10],
   yourLocation: 'your location',
+  placeSearch: 'place',
+  postcodeSearch: 'postcode'
```

Comment:
Can this line end in a comma please as it make future changes easier to see. There is a rule in eslint for [comma dangle](https://eslint.org/docs/rules/comma-dangle) that we should check the status of as I thought it was setup to be on but as it hasn't picked this up it suggests the rule isn't active.